### DISCUSSION TITLE
Is @mantine/next required with v7

### DISCUSSION THREAD

**Discussion by matt-greenfield-15** [mantinedev/mantine#4807] [Q&A] [answered]
I've noted that https://www.npmjs.com/package/@mantine/next does not have a v7 release. Does that mean I should remove the package when upgrading to v7?
I know that it used to server rendering emotionCSS so it makes sense that we don't need it anymore.

If so, should this be mentioned in the upgrade guide here? https://mantine.dev/guides/6x-to-7x

Thanks

**Comment by rtivital** [comment, marked as answer]
Yes, this package is no longer required if you use mantine with Next.js, you can safely remove it.

**Reply by matt-greenfield-15** [reply]
thank you
